modules/dnn/test/test_darknet_importer.cpp

```diff
@@ -528,8 +528,12 @@ TEST_P(Test_Darknet_layers, reorg)
     testDarknetLayer("reorg");
 }
 
-TEST_P(Test_Darknet_layers, convolutional )
+TEST_P(Test_Darknet_layers, convolutional)
 {
+    if (target == DNN_TARGET_MYRIAD)
+    {
+        default_l1 = 0.01f;
+    }
```

Comment:
Do we have different objects of `Test_Darknet_layers` class for each test case? Just to check that `default_l1` will be different after this test.

Reply:
Each test case has own instance (state) of test object (for each parameter too).

Reply:
Got it, thanks!